lib/active_admin/engine.rb

```diff
@@ -8,5 +8,9 @@ class Engine < ::Rails::Engine
         app.config.assets.precompile << path
       end
     end
+
+    initializer 'active_admin.routes' do
+      require 'active_admin/helpers/routes/url_helpers'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.